client/src/components/page/Header/index.tsx

```diff
@@ -58,7 +59,7 @@ export const HeaderComponent: React.FC<HeaderProps> = (props: HeaderProps) => {
           <Menu.Item key={URL_HOME}>
             <Link to={URL_HOME}>Home</Link>
           </Menu.Item>
-          {REACT_APP_LANGUAGE_SWITCHER && (
+          {(process.env.REACT_APP_LANGUAGE_SWITCHER === 'true' || REACT_APP_LANGUAGE_SWITCHER) && (
```

Comment:
Why do we need to have this both here and `configs/env.ts`?  Can we find a way to just import something from `configs/env.ts` and use that?  The idea is to try and have all the configs in one place - and right now that place is `configs/env.ts`

Reply:
tests weren't working and to get the process.env at run time during tests i have to add this extra condition.

if i dont do this the coverage will be reduce when language switcher is set to false

Reply:
@ashfahan I think instead of that we can [mock the envs](https://github.com/OpenSRP/web/tree/master/client/src/configs/__mocks__), no?

Reply:
i did mock them but the component doesn't seem to update as per the mocked one when imported from env file unless i applied that hack. https://github.com/OpenSRP/web/blob/310aa4941046f3c9a8b8c622df7f265b24816ed6/client/src/containers/ConnectedHeader/tests/index.test.tsx#L50-L65

Reply:
@p-netm @kelvin-muchiri how do you feel about this?

Reply:
@ashfahan I think we should have all configs initialized and imported from the same place which is `configs/envs.ts`.
I hope [this] this will be helpful in terms of mocking.

Reply:
@p-netm @moshthepitt 

this might help me to better explain the problem i am facing right now
![image](https://user-images.githubusercontent.com/19289015/97149553-b9818480-178e-11eb-848d-6865494e1810.png)

the left one is the code i wrote into the component and middle one is the console of browser and right one is the console of the jest test

when i import any environment variable from the file the just simply resolve to undefined
but when i fetch them using process.env they resolve to their actual  values this is the reason why i included extra check in here https://github.com/OpenSRP/web/blob/f2732184ff54a0ced1d489b5aaf34ba11719d405/client/src/components/page/Header/index.tsx#L62

cc: @kelvin-muchiri

Reply:
> @ashfahan I think we should have all configs initialized and imported from a single source which is `configs/envs.ts`.
> I hope [this] this will be helpful in terms of mocking.

forgot to add this link: https://jestjs.io/docs/en/manual-mocks#mocking-user-modules

Reply:
https://github.com/onaio/reveal-frontend/blob/fab106acd078fe4f991006d7006154b69647bf55/src/containers/pages/InterventionPlan/PlanDefinitionList/tests/index.test.tsx#L27

Reply:
@ashfahan Could it have to do with the typo I see in your screenshot?

Reply:
> @ashfahan Could it have to do with the typo I see in your screenshot?

no it have nothing to do with the typo

Reply:
mock testing is working fine on local but how can we get it past circle ci as .env arent pushed to git and thus are unavailable inside circle ci.

![image](https://user-images.githubusercontent.com/19289015/97874978-9aa96200-1d3b-11eb-871c-11f6e93a4a1a.png)